tools/publish.sh

```diff
@@ -11,6 +11,7 @@ crates=(
     bevy_reflect
     bevy_ecs/macros
     bevy_ecs
+    bevy_state
```

Comment:
`bevy_state_macros` will need to be added as well.

Reply:
Thanks! I can't believe I forgot that

Reply:
Thanks! I can't believe I forgot that!